paddle/fluid/platform/cpu_info.cc

```diff
@@ -28,9 +28,13 @@ DEFINE_double(fraction_of_cpu_memory_to_use, 1,
               "Default use 100% of CPU memory for PaddlePaddle,"
               "reserve the rest for page tables, etc");
 
-DEFINE_uint64(
-    initial_cpu_memory_in_mb, 500,
-    "Default initial 500MB of CPU memory for PaddlePaddle, in MD unit.");
+DEFINE_uint64(initial_cpu_memory_in_mb,
+#ifdef PADDLE_WITH_MKLDNN
+              1000,
```

Comment:
Add some comments to tell why 1000 works.

Reply:
Actually, I am not sure why this exactly 1000 number may works.

I was trying to give more space, here gives 2x of original, for MKLDNN according to the comment from mozga-intel.

And that's why I add @mozga-intel to help to review.

Please @mozga-intel, could you please help check if is this enough.

BTW, I add this flag to env, so users can change this from env if this default 1000MB were not enough.

Reply:
@tensor-tang Of course.

Reply:
@tensor-tang IMO I think that 1000MB of space of memory is too small. The good result is obtained when the memory is greater than 5000 MB

Reply:
OK, thanks @mozga-intel, I will replace it with 5000 MB.